compendium/modules/w08-matrices-exercise.tex

```diff
@@ -81,16 +81,16 @@
 \Subtask Skapa nedan heltalsmatris i REPL. Vilken dimension får matrisen?
 \begin{REPL}
 val ds1 = for (i <- 1 to 1000) yield {
-            for (j <- 1 to 5) yield throwDie
+            for (j <- 1 to 5) yield throwDice
```

Comment:
Why that? So that it means "kasta dö"? Or does Die mean something else in computer language?